src/sentry/middleware/integrations/parsers/github.py

```diff
@@ -60,6 +60,13 @@ def try_forward_to_codecov(self, event: Mapping[str, Any]) -> None:
             metrics.incr("codecov.forward-webhooks.forward-error", sample_rate=0.01)
 
     def get_response(self) -> HttpResponseBase:
+        """
+        Orchestrates GitHub webhook routing across Sentry's multi-service architecture.
+
+        Handles installation events in control silo, distributes webhooks to appropriate
+        region silos based on organization locations, and conditionally forwards to
+        external services (Codecov, Overwatch) based on configuration and region.
```

Comment:
At some point, we will stop calling Overwatch. This is the middleware layer where there's controlling of which region to send the webhook to.

The call is piggybacked at the end of this function:
https://github.com/getsentry/sentry/blob/de9de84d753ab55d95dddd03051bf130da1d8032/src/sentry/middleware/integrations/parsers/github.py#L100-L105